docs/import/create_score_asset.js

```diff
@@ -467,16 +570,16 @@ function calculateDamage(assetData, setMapBoundsInfo) {
  * This method will go away or be greatly changed if we're using CrowdAI data
  * instead of previously computed building data.
  * @param {string} buildingPath location of buildings asset in EE
- * @param {ee.FeatureCollection} stateGroups Collection with block groups
- * @return {ee.Dictionary} Number of buildings per block group
+ * @param {ee.FeatureCollection} geographies Collection with districts
```

Comment:
nit: maybe s/districts/geographical areas since districts is an actual census thing

Reply:
Can you point me to what "district" means in the census context? I didn't think it was a special word. There used to be an "enumeration district", but I think it's obsolete, and anyway "block group" is the modern version of "enumeration district", so it seems appropriate.

Reply:
Just stuff like this: https://www.census.gov/mycd/

Reply:
Yeah, that seems far enough removed from actual Census definitions that it seems unlikely to confuse. We're going to need a generic word for "block groups" for this work, and "district" seems better than "geographical areas". If you can think of another succinct generic term, let me know!